css/properties/cursor.json

```diff
@@ -658,12 +658,12 @@
             ],
             "support": {
               "chrome": {
-                "version_added": "≤83"
+                "version_added": "68"
```

Comment:
I tested and determined that we can copy the support statement from `grab` (verified that `-webkit-grab` works in Chrome 22 on Windows via BrowserStack Live):

(Although we might want to remove the unnecessary note on the unprefixed version.)

```
              "chrome": [
                {
                  "version_added": "68",
                  "notes": "Chrome also continues to support the prefixed versions."
                },
                {
                  "prefix": "-webkit-",
                  "version_added": "1",
                  "notes": "Chrome 22 added Windows support."
                }
              ],
```